src/compas_ui/rhino/forms/scene.py

```diff
@@ -70,7 +70,7 @@ def OnCreateCell(self, args):
             def on_click(sender, e):
                 form = SettingsForm(obj.settings)
                 if form.ShowModal(self.parent):
-                    obj.settings.update(form.settings)
```

Comment:
I think we can skip this step now, since with the strong type control of Value Objects, we can safely mutate original Settings object when user press OK on the form without making a copy in the background.